buildscripts/ivy.xml

```diff
@@ -1,4 +1,4 @@
-<ivy-module version="2.0" xmlns:e="http://ant.apache.org/ivy/extra">
+ <ivy-module version="2.0" xmlns:e="http://ant.apache.org/ivy/extra">
```

Comment:
Sorry this feels like nitpicking, but would appreciate if you could remove this spurious space change.

Reply:
done